meshtastic/mesh.proto

```diff
@@ -901,8 +901,8 @@ message MeshPacket {
   fixed32 from = 1;
 
   /*
-   * The (immediatSee Priority description for more details.y should be fixed32 instead, this encoding only
-   * hurts the ble link though.
+   * The (immediate) destination for this packet. If we are using routing, the
+   * final destination will be in payload.dest
```

Comment:
Not sure if we should mention `payload.dest` here. This was added when an attempt was made to do multi-hop routing, but it is currently unused. The firmware never sets `dest` and doesn't use it either, same for clients as far as I know.

Reply:
Maybe we can even mark `dest` and `source` in the payload as deprecated if confirmed no clients use it.

Reply:
It's at least better than what it currently is :) 
But while we're here probably better to remove the confusing part about `payload.dest`.

Reply:
Removed the reference in [9515cf2](https://github.com/meshtastic/protobufs/pull/450/commits/9515cf2fb5d5e4e375bb6b11809817e439743ef3)